updateAnnouncements.py

```diff
@@ -0,0 +1,52 @@
+# Initial function:
+# Inputs: Github API credentials (not using for now)
+# call the github API to get all the comments for a certain issue on the hoth github
+# format the data into a json containing the data that we want
+# inject the json into a json file
+
+import requests
+import json
+import sys
+import os
+
+# User whitelist and issue number we are looking for
+user_whitelist = ['jodymlin', 'khxia', 'jakobreinwald', 'christinatong01', 'milesswu']
+# HOTH repository issues accessed via Github API
+repo_issues = "https://api.github.com/repos/uclaacm/hoth.uclaacm.com/issues"
+
+issue_num = '181'
+
+url = repo_issues + '/' + issue_num + '/' + 'comments'
+
+# Get given issue
+try:
+    announcements = requests.get(url)
+except requests.exceptions.RequestException as e:
+    raise SystemExit(e)
+print("Request completed with status code: ", announcements.status_code)
+
+# Convert announcements from Github API to json for parsing
+try:
+    announcements_json = announcements.json()
+except:
+    print("Error obtaining json from Github API return. ")
+valid_comments = []
+
+# Keep track of every comment from valid usernames
+try:
```

Comment:
Let's remove the try-catch here, and let the program fail on its own when we encounter an error.